### PR TITLE
Add readme with generation model description

### PR DESCRIPTION
**Pull request by avan1235**
small fixes after merge mess

### CODE REVIEW

File: android/app/src/main/res/layout/activity_hello_select.xml [hunk 1]
```diff
@@ -8,20 +8,6 @@
     android:layout_height="match_parent"
     tools:context=".ui.activity.hello.HelloSelectActivity">
 
-    <ImageView
```
Comment (kantoniak) [inline android/app/src/main/res/layout/activity_hello_select.xml:11]: By accident?

File: android/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Android client application
+
+Android app is written with Kotlin language as the client endpoint for EatersLab system. App allows users to search for nearby or favourite canteens and check occupancy data and other available information for cafeterias.
+App is designed for min SDK 21 version so Android Lollipop or newer devices are supported (so really few devices are unsupported)
+
+## App building instructions
+
+App is developed in AndroidStudio and for development should be imported as standard Android app project. Before building the project and indexing files in IDE the generation process is needed.
+
+### Client model generation
```
Comment (kantoniak) [inline android/README.md:10]: I would use "API" in this heading so it's easier to find for people who look for API generation.

File: android/app/src/main/res/layout/activity_hello_select.xml (hunk 1 above)
Comment (avan1235) [reply android/app/src/main/res/layout/activity_hello_select.xml:11]: yep

File: android/README.md (hunk 2 above)
Comment (avan1235) [reply android/README.md]: fixed

**Pull request merged**